libctru/source/services/ac.c

```diff
@@ -16,8 +16,9 @@ Result acInit(void)
 
 	if (AtomicPostIncrement(&acRefCount)) return 0;
 
-	ret = srvGetServiceHandle(&acHandle, "ac:u");
-	if(R_FAILED(ret)) ret = srvGetServiceHandle(&acHandle, "ac:i");
+	// ac:i has the most commands, then ac:u
```

Comment:
Is this really needed? Wiki only lists 4XX cmds as ac:i exclusive.

Reply:
that was requested by someone else, and since it happens with cfg why not with ac